test/src/API/projects/watchList-fileChanges.test.js

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
```

Comment:
Was this moved from `test/src/NOTRUNNING/projects/watchList-fileChanges.test.js` should it have been done with `git mv` to keep the history and show changes?

Reply:
I don't think I used `git mv` (I used the VSCode git UI) but in any case there were significant changes so I thought it best to review this test file as new :)